tasks/exim4.yml

```diff
@@ -48,6 +48,13 @@
   notify:
     - restart exim4
 
+- name: Copy sender address rewriting config
+  template:
+    src: 32_exim4-sender_rewriting.j2
+    dest: /etc/exim4/conf.d/rewrite/32_exim4-sender_rewriting
```

Comment:
I wonder in what order should the rewrite be done? Currently, there was an entry with prefix number 31 so I simply added one to that so that this rewrite is done afterwards.